package.json

```diff
@@ -20,8 +20,8 @@
     "lodash": "^4.17.20",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-paginate": "^6.5.0",
-    "react-scripts": "^5.0.1",
+    "react-paginate": "^8.2.0",
+    "react-scripts": "^2.1.3",
```

Comment:
@gonziesc 
1. Why do we downgrade version for react-scripts?
2. Can we update node-version to `18.6.0` in `.nvmrc` since the current versions are being [deprecated by Vercel](https://vercel.com/changelog/node-js-14-and-16-are-being-deprecated)?